Makefile

```diff
@@ -43,8 +54,13 @@ clean:					#: Delete all build artifacts
 	rm -rf $(BUILD_BASE_DIR)
 
 cmake:					#: Use CMake to create a Makefile build system
-	mkdir -p $(BUILD_BASE_DIR)/$(BUILD_DIR) && \
-	cmake -B "$(BUILD_BASE_DIR)/$(BUILD_DIR)" $(GENERATOR) $(USE_CCACHE) $(FORCE_COLOR) ${WARNINGS_AS_ERRORS} ${ENABLE_ALL_WARNINGS} -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+	mkdir -p $(BUILD_BASE_DIR)/$(BUILD_DIR)
```

Comment:
the mkdir is actually redundant; cmake will create the binary directory automatically.